repository.py

```diff
@@ -254,6 +254,8 @@ def __init__(self, accessor):
         super(MainYumRepository, self).__init__(accessor)
         self._identifier = MAIN_REPOSITORY_NAME
         self.keyfiles = []
+        self.__gpg_check = True
+        self.__repo_gpg_check = True
```

Comment:
Seems to me the convention in this project is to use a single "_" for private attributes.

Reply:
I agree we should use a single underscore here, but because they are not "attributes that you do not want subclasses to use" (PEP8 wording)